### DISCUSSION TITLE
Can't download anything from Github on my mac.

### DISCUSSION THREAD

**Discussion by kosbarts** [community/community#125610] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello. Out of the blue my Mac has stopped downloading files from Github. 

I can't Download any zip from any repo (download is just hanging...)
I can't install anything over Terminal.
I can't download from any repo via an app (i.e. Glyphs App can't install its modules from Github repos)

Other computers on the same network do not have any issues.

Any help will be much appreciated.

**Comment by kosbarts** [comment]
So the problem is solved. It was some Glyphs download that was left hanging in the background in some folder and once I uninstalled Glyphs everything is working fine.

**Reply by abba-m** [reply]
Hi
How did you check to see what was hanging (in this case the Glyphs)

**Reply by kosbarts** [reply]
I think it was suggested by someone on the Glyphs forum. I am sorry but it's been a while so I've completely forgotten the details now.
